### ISSUE TITLE
Version check and release notes in Console

### ISSUE THREAD

**Issue by johanstokking** [TheThingsNetwork/lorawan-stack#10]
**Summary:** 
- We should have release notes for new versions
- These release notes cannot be linked to the public repository GitHub releases, as some functionality is in this private repository
- We should make it available easily, I would propose the Console

**Why do we need this?**
It makes sense to use the console for this as it is where most of the people this would concern should be. Also then it's available in any deployment model (i.e. on-premises and SaaS)

**What is already there? What do you see now?**
- The release notes (extracted from our issues)

**What is missing? What do you want to see?**
- A screendesign and concept for where the release notes should appear
- Can we fetch GitHub releases and their notes from this private repository in Markdown format and display them in the Console? cc @htdvisser @kschiffer

See also #10

----

Original issue: https://github.com/TheThingsIndustries/lorawan-stack/issues/607 by @johanstokking

**Comment by johanstokking** [comment]
Let's get this from the GitHub release as generated by goreleaser.

**Comment by htdvisser** [comment]
Release notes for the current version would be on https://api.github.com/repos/TheThingsNetwork/lorawan-stack/releases/tags/v3.0.0 (or similar), but I can imagine it would also be very helpful to print the latest N releases from https://api.github.com/repos/TheThingsNetwork/lorawan-stack/releases

**Comment by nejraselimovic** [comment]
@kschiffer are we planning to have the release notes in the Console too (besides documentation page)?

**Comment by johanstokking** [comment]
References https://github.com/TheThingsIndustries/lorawan-stack-aws-integration/issues/29

**Comment by kschiffer** [comment]
> [**@kschiffer**](https://github.com/kschiffer) are we planning to have the release notes in the Console too (besides documentation page)?

Yes, this is planned. See TheThingsIndustries/lorawan-stack#2663

**Comment by kschiffer** [comment]
> [**@kschiffer**](https://github.com/kschiffer) are we planning to have the release notes in the Console too (besides documentation page)?

Yes, this is planned. See TheThingsIndustries/lorawan-stack#2663

**Comment by nejraselimovic** [comment]
> > [**@kschiffer**](https://github.com/kschiffer) are we planning to have the release notes in the Console too (besides documentation page)?
> 
> Yes, this is planned. See [TheThingsIndustries/lorawan-stack#2663](https://github.com/TheThingsIndustries/lorawan-stack/issues/2663)

Great, I'm moving this to Q3 then.

**Comment by johanstokking** [comment]
References https://github.com/TheThingsIndustries/lorawan-stack/pull/2759

**Comment by johanstokking** [comment]
Release notes are now in the documentation.

Version check comes, first and foremost in logs. We may want to show new version availability in the Console as well, but that isn't too important. Removing milestone.

**Comment by KrishnaIyer** [comment]
This is all done. We have release notes (both TTS and AWS) and version checks.
